<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="HttpException.xml" source-language="en-US" target-language="ko-KR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac51864e8fe52f0eb87b7c23fbfe505d15cf9681ede.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1864e8fe52f0eb87b7c23fbfe505d15cf9681ede</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Web.HttpException">
          <source>Describes an exception that occurred during the processing of HTTP requests.</source>
          <target state="translated">HTTP 요청을 처리하는 동안 발생한 예외를 설명합니다.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> class is an HTTP-specific exception class that enables ASP.NET to generate exception information.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 클래스는 ASP.NET에서 예외 정보를 생성할 수 있도록 하는 HTTP 관련 예외 클래스입니다.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>For more information about throwing and handling exceptions, see <bpt id="p1">[</bpt>Exceptions<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>.</source>
          <target state="translated">throw 한 예외를 처리 하는 방법에 대 한 자세한 내용은 참조 <bpt id="p1">[</bpt>예외<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>합니다.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The following code example demonstrates how to use the <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> class to raise customized exceptions.</source>
          <target state="translated">다음 코드 예제에서는 사용 하는 방법을 보여 줍니다.는 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 클래스를 사용자 지정 된 예외를 발생 합니다.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The <ph id="ph1">`Button_Click`</ph> method determines which button was clicked, and then calls either the <ph id="ph2">`CheckNumber`</ph> or <ph id="ph3">`CheckBoolean`</ph> method.</source>
          <target state="translated"><ph id="ph1">`Button_Click`</ph> 메서드 확인 단추를 클릭 한 한 다음 중 하나를 호출는 <ph id="ph2">`CheckNumber`</ph> 또는 <ph id="ph3">`CheckBoolean`</ph> 메서드.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>If the user-entered value in the text box does not correspond to the expected type, an <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> exception is created and thrown with a customized message based on the error type.</source>
          <target state="translated">필요한 형식으로 텍스트 상자에 사용자가 입력 한 값을 해당 하지 않는 경우는 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 예외가 생성 되 고 오류 유형을 기준으로 사용자 지정 된 메시지와 함께 발생 합니다.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="T:System.Web.HttpException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class and creates an empty <ph id="ph2">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> object.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화하여 빈 <ph id="ph2">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 개체를 만듭니다.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">예외를 처리할 때는 일련의 내부 예외에 대 한 응답에서 throw 되는 외부 예외와 관련 된 예외를 캡처하려면 유용할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">외부 예외를 발생 시킨 내부 예외에 대 한 참조는에서 사용할 수는 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 외부 예외의 속성입니다.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">이 메커니즘 보다 의미 있는 외부 예외를 만들 수 있도록 하는 동안 원래 예외를 포함 하는 이전 예외에 의해 수행 되는 오류 정보를 유지 합니다.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">자세한 내용은 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>을 참조하세요.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">다음 코드 예제는 <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 의 생성자는 <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> 클래스입니다.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value through a text box and checks whether it is an integer.</source>
          <target state="translated"><ph id="ph1">`CheckNumber`</ph> 메서드 텍스트 상자를 통해 사용자가 입력 한 값을 허용 하 고 정수 인지 확인 합니다.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>If the value is not an integer, an exception is thrown, and then a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">정수 값 없는 경우 예외가 throw 됩니다, 한 다음 새 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 개체가 생성 되 고 throw 됩니다.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message is displayed on the browser.</source>
          <target state="translated">예외에서 발견 되는 <ph id="ph1">`Button_Click`</ph> 이벤트 처리기 및 오류 메시지는 브라우저에 표시 됩니다.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using a supplied error message.</source>
          <target state="translated">제공된 오류 메시지를 사용하여 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">다음 코드 예제는 <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 의 생성자는 <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> 클래스입니다.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>An <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> is thrown if a user-entered value is 0.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 사용자가 입력 한 값이 0 이면 throw 됩니다.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>The HTTP response status code sent to the client corresponding to this error.</source>
          <target state="translated">이 오류에 해당하는 클라이언트로 보낸 HTTP 응답 상태 코드입니다.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code and an error message.</source>
          <target state="translated">HTTP 응답 상태 코드 및 오류 메시지를 사용하여 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">throw되는 예외에 대해 serialize된 개체 데이터를 보유하는 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 개체입니다.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that holds the contextual information about the source or destination.</source>
          <target state="translated">소스 또는 대상에 대한 컨텍스트 정보를 보유하는 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">serialize된 데이터를 사용하여 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor is called during deserialization to reconstitute the exception object that is transmitted over a stream.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 생성자는 역직렬화 스트림을 통해 전송 되는 예외 개체를 다시 구성 하는 동안 호출 되어 합니다.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">자세한 내용은 참조 <bpt id="p1">[</bpt>XML 및 SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>합니다.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, if any, that threw the current exception.</source>
          <target state="translated">현재 예외를 throw하는 <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>(있을 경우)입니다.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an error message and the <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">오류 메시지와 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 속성을 사용하여 <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">예외를 처리할 때는 일련의 내부 예외에 대 한 응답에서 throw 되는 외부 예외와 관련 된 예외를 캡처하려면 유용할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">외부 예외를 발생 시킨 내부 예외에 대 한 참조는에서 사용할 수는 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 외부 예외의 속성입니다.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">이 메커니즘 보다 의미 있는 외부 예외를 만들 수 있도록 하는 동안 원래 예외를 포함 하는 이전 예외에 의해 수행 되는 오류 정보를 유지 합니다.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">자세한 내용은 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>을 참조하세요.</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">다음 코드 예제는 <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 의 생성자는 <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> 클래스입니다.</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value through a text box and checks whether it is an integer.</source>
          <target state="translated"><ph id="ph1">`CheckNumber`</ph> 메서드 텍스트 상자를 통해 사용자가 입력 한 값을 허용 하 고 정수 인지 확인 합니다.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>If the value is not an integer, an exception is thrown, and then in the catch block, a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">정수 값 없는 경우 예외가 throw 됩니다, 한 다음 catch 블록에서 새 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 개체가 생성 되 고 throw 됩니다.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message is displayed on the browser.</source>
          <target state="translated">예외에서 발견 되는 <ph id="ph1">`Button_Click`</ph> 이벤트 처리기 및 오류 메시지는 브라우저에 표시 됩니다.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The exception code that defines the error.</source>
          <target state="translated">오류를 정의하는 예외 코드입니다.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an error message and an exception code.</source>
          <target state="translated">오류 메시지와 예외 코드를 사용하여 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">다음 코드 예제는 <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 의 생성자는 <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> 클래스입니다.</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>An <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> exception is thrown if a user-entered value is 0.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 사용자가 입력 한 값이 0 이면 예외가 발생 합니다.</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="160" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The HTTP response status code displayed on the client.</source>
          <target state="translated">클라이언트에 표시되는 HTTP 응답 상태 코드입니다.</target>       </trans-unit>
        <trans-unit id="161" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="162" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, if any, that threw the current exception.</source>
          <target state="translated">현재 예외를 throw하는 <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>(있을 경우)입니다.</target>       </trans-unit>
        <trans-unit id="163" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code, an error message, and the <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">HTTP 응답 상태 코드, 오류 메시지 및 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 속성을 사용하여 <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="164" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">예외를 처리할 때는 일련의 내부 예외에 대 한 응답에서 throw 되는 외부 예외와 관련 된 예외를 캡처하려면 유용할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="165" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">외부 예외를 발생 시킨 내부 예외에 대 한 참조는에서 사용할 수는 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 외부 예외의 속성입니다.</target>       </trans-unit>
        <trans-unit id="166" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">이 메커니즘 보다 의미 있는 외부 예외를 만들 수 있도록 하는 동안 원래 예외를 포함 하는 이전 예외에 의해 수행 되는 오류 정보를 유지 합니다.</target>       </trans-unit>
        <trans-unit id="167" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">자세한 내용은 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>을 참조하세요.</target>       </trans-unit>
        <trans-unit id="168" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">다음 코드 예제는 <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 의 생성자는 <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> 클래스입니다.</target>       </trans-unit>
        <trans-unit id="169" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value and checks whether it is an integer.</source>
          <target state="translated"><ph id="ph1">`CheckNumber`</ph> 메서드는 사용자가 입력 한 값을 허용 하 고 정수 인지 확인 합니다.</target>       </trans-unit>
        <trans-unit id="170" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>If the value is not an integer, an exception is thrown, and then a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object containing the HTTP response status code, the exception's message, and any inner exception is created.</source>
          <target state="translated">값 정수 없으면 예외가 throw 됩니다, 한 다음 새 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> HTTP 응답 상태 코드, 예외 메시지 및 내부 예외를 모두 포함 된 개체가 만들어집니다.</target>       </trans-unit>
        <trans-unit id="171" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message, error code, and inner exception are displayed.</source>
          <target state="translated">예외에서 발견 되는 <ph id="ph1">`Button_Click`</ph> 이벤트 처리기 및 오류 메시지, 오류 코드 및 내부 예외를 표시 됩니다.</target>       </trans-unit>
        <trans-unit id="172" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="173" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="174" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="175" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The HTTP response status code displayed on the client.</source>
          <target state="translated">클라이언트에 표시되는 HTTP 응답 상태 코드입니다.</target>       </trans-unit>
        <trans-unit id="176" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="177" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The exception code that defines the error.</source>
          <target state="translated">오류를 정의하는 예외 코드입니다.</target>       </trans-unit>
        <trans-unit id="178" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code, an error message, and an exception code.</source>
          <target state="translated">HTTP 응답 상태 코드, 오류 메시지 및 예외 코드를 사용하여 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="179" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">다음 코드 예제는 <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> 의 생성자는 <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> 클래스입니다.</target>       </trans-unit>
        <trans-unit id="180" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The user name and email information are entered by the user in the provided text boxes.</source>
          <target state="translated">사용자 이름 및 전자 메일 정보는 제공 된 텍스트 상자에 사용자가 입력 됩니다.</target>       </trans-unit>
        <trans-unit id="181" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>If any of the text boxes are left blank, an <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">텍스트 상자는 비워 둘 경우는 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 개체가 생성 되 고 throw 됩니다.</target>       </trans-unit>
        <trans-unit id="182" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The error code of the <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> is obtained by the <ph id="ph2">&lt;xref:System.Web.HttpException.GetHttpCode%2A&gt;</ph> method and displayed on the Web page.</source>
          <target state="translated">오류 코드는 <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> 여 가져온는 <ph id="ph2">&lt;xref:System.Web.HttpException.GetHttpCode%2A&gt;</ph> 메서드 웹 페이지에 표시 합니다.</target>       </trans-unit>
        <trans-unit id="183" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">이 예제에는 사용자 입력을 허용하는 텍스트 상자가 있으므로 보안상 위험할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="184" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">기본적으로 ASP.NET 웹 페이지는 사용자 입력 내용에 스크립트 또는 HTML 요소가 포함되어 있지 않은지 확인합니다.</target>       </trans-unit>
        <trans-unit id="185" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">자세한 내용은 <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>를 참조하세요.</target>       </trans-unit>
        <trans-unit id="186" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">예외가 throw될 때 클라이언트에 표시되는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="187" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>Creates a new <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> exception based on the error code that is returned from the Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> method.</source>
          <target state="translated">Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> 메서드에서 반환되는 오류 코드를 기반으로 하는 새 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> 예외를 만듭니다.</target>       </trans-unit>
        <trans-unit id="188" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>An <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> based on the error code that is returned from a call to the Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> method.</source>
          <target state="translated">Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> 메서드에 대한 호출에서 반환되는 오류 코드를 기반으로 하는 <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="189" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHtmlErrorMessage">
          <source>Gets the HTML error message to return to the client.</source>
          <target state="translated">HTML 오류 메시지를 가져와 클라이언트에게 반환합니다.</target>       </trans-unit>
        <trans-unit id="190" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHtmlErrorMessage">
          <source>The HTML error message.</source>
          <target state="translated">HTML 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="191" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHttpCode">
          <source>Gets the HTTP response status code to return to the client.</source>
          <target state="translated">HTTP 상태 응답 코드를 가져와 클라이언트에게 반환합니다.</target>       </trans-unit>
        <trans-unit id="192" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHttpCode">
          <source>A non-zero HTTP code representing the exception or the <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> code; otherwise, HTTP response status code 500.</source>
          <target state="translated">예외 또는 <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> 코드를 나타내는 0이 아닌 HTTP 코드이거나, 그렇지 않은 경우 HTTP 응답 상태 코드 500입니다.</target>       </trans-unit>
        <trans-unit id="193" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>If there is a non-zero HTTP code, it is returned.</source>
          <target state="translated">0이 아닌 HTTP 코드 인 경우 반환 됩니다.</target>       </trans-unit>
        <trans-unit id="194" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>Otherwise, the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> code is returned.</source>
          <target state="translated">그렇지 않은 경우는 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 코드가 반환 됩니다.</target>       </trans-unit>
        <trans-unit id="195" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>If neither an <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> code nor a non-zero HTTP code is available, the HTTP response status code 500 is returned.</source>
          <target state="translated">모두는 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 코드나 0이 아닌 HTTP 코드를 사용할 수 있는 500 HTTP 응답 상태 코드가 반환 됩니다.</target>       </trans-unit>
        <trans-unit id="196" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>For more information on HTTP response status codes, see the Status Code Definitions of the Hypertext Transfer Protocol - HTTP/1.1 on the World <bpt id="p1">[</bpt>Wide Web Consortium (W3C) Web site<ept id="p1">](http://go.microsoft.com/fwlink/?linkid=37125)</ept>.</source>
          <target state="translated">HTTP 응답 상태 코드에 대 한 자세한 내용은 참조는 Hypertext Transfer Protocol-전 세계에서 HTTP/1.1 상태 코드 정의 <bpt id="p1">[</bpt>Wide Web Consortium (W3C) 웹 사이트<ept id="p1">](http://go.microsoft.com/fwlink/?linkid=37125)</ept>합니다.</target>       </trans-unit>
        <trans-unit id="197" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">throw되는 예외에 대해 serialize된 개체 데이터를 보유하는 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 개체입니다.</target>       </trans-unit>
        <trans-unit id="198" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that holds the contextual information about the source or destination.</source>
          <target state="translated">소스 또는 대상에 대한 컨텍스트 정보를 보유하는 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="199" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Gets information about the exception and adds it to the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object.</source>
          <target state="translated">예외에 대한 정보를 가져와 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 개체에 추가합니다.</target>       </trans-unit>
        <trans-unit id="200" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException.GetObjectData%2A&gt;</ph> method adds information about the exception to the <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> object that is targeted for serialization.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpException.GetObjectData%2A&gt;</ph> 에 대 한 예외에 대 한 정보를 추가 하는 메서드는 <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> 개체 serialization에 대 한 대상입니다.</target>       </trans-unit>
        <trans-unit id="201" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During deserialization, the exception is reconstituted from the <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> that is transmitted over the stream.</source>
          <target state="translated">deserialization을 수행 하는 동안 예외가 재구성 됩니다는 <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> 스트림을 통해 전송 되는 합니다.</target>       </trans-unit>
        <trans-unit id="202" translate="yes" xml:space="preserve" uid="P:System.Web.HttpException.WebEventCode">
          <source>Gets the event codes that are associated with the HTTP exception.</source>
          <target state="translated">HTTP 예외와 연결된 이벤트 코드를 가져옵니다.</target>       </trans-unit>
        <trans-unit id="203" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpException.WebEventCode">
          <source>An integer representing a Web event code.</source>
          <target state="translated">웹 이벤트 코드를 나타내는 정수입니다.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>